### ISSUE TITLE
[Enhancement]Fully customisable perks 

### ISSUE THREAD

**Issue by Willstuff1111** [PavelZinchenko/event-horizon-main#437]
Here are some examples:

> Duplicateable perk files 
> Adtional modifiers such as speed, turn rate, corrosive resistance
> Changeable star costs for perks
> Ability to add a feature to perks(a plus button that would add select modifiers to perk file or something)

**Comment by Willstuff1111** [comment]
Another one is perks adding an inbuilt device, like a repair drone, or a cloning system, or even a weapon

**Comment by SuplexCity25284** [comment]
I agree that resisto corrosive damage needs to be added, even if as a perk.  I'd want the same ability in Frontier as well
